packages/io/src/io.ts

```diff
@@ -227,7 +227,7 @@ export async function which(tool: string, check?: boolean): Promise<string> {
     // build the list of directories
     //
     // Note, technically "where" checks the current directory on Windows. From a task lib perspective,
-    // it feels like we should not do this. Checking the current directory seems like more of a use
+    // it feels like we should not do this. Check ing the current directory seems like more of a use
```

Comment:
I don't think we need this change.

Reply:
There's also now conflicts from separate changes - you can probably just resolve using the version from master

Reply:
Sorry, global search-and-replace was trying to change `Checkin` not `Checking` but was careless.